mathesar/api/serializers/constraints.py

```diff
@@ -129,3 +151,15 @@ def get_mapping_field(self, data):
         else:
             constraint_type = data.get('type', None)
         return constraint_type
+
+    def run_validation(self, data):
+        print(f'context === {self.context}')
+        columns = data.get('columns', None)
+        if columns == []:
```

Comment:
Any reason why you are not setting the parameter `allow_empty=False` to the field for the empty value check

Reply:
Since the `columns` parameter is a primary key-related field instead of a List field, it does not allow to have `allow_empty=False`. 
See: https://www.django-rest-framework.org/api-guide/relations/#primarykeyrelatedfield